### PR TITLE
negar/update_charting_page

### PR DESCRIPTION
**Pull request by negar-binary**
update charting logos, fix mobile and tablet view

### CODE REVIEW

File: src/images/pages/charting/tradingview-g-app.svg [hunk 1]
```diff
@@ -1,179 +1 @@
...
...51.64" width="87.73" height="87.73"/><path class="cls-2" d="M138.73,52.27v86.46H52.27V52.27h86.46M140,51H51v89h89V51h0Z"/><line class="cls-7" x1="62.55" y1="140" x2="62.55" y2="51"/><line class="cls-7" x1="75.73" y1="140" x2="75.73" y2="51"/><line class="cls-7" x1="88.91" y1="140" x2="88.91" y2="51"/><line class="cls-7" x1="102.09" y1="140" x2="102.09" y2="51"/><line class="cls-7" x1="115.27" y1="140" x2="115.27" y2="51"/><line class="cls-7" x1="128.45" y1="140" x2="128.45" y2="51"/><line class="cls-7" x1="51" y1="62.55" x2="140" y2="62.55"/><line class="cls-7" x1="51" y1="75.73" x2="140" y2="75.73"/><line class="cls-7" x1="51" y1="88.91" x2="140" y2="88.91"/><line class="cls-7" x1="51" y1="102.09" x2="140" y2="102.09"/><line class="cls-7" x1="51" y1="115.27" x2="140" y2="115.27"/><line class="cls-7" x1="51" y1="128.45" x2="140" y2="128.45"/><polyline class="cls-8" points="54.92 102.43 62.55 109.82 73.38 90.41 78.54 92.78 82.39 105.85 88.91 98.05 98.6 102.28 102.09 104.92 113.08 95.93 117.53 84.99 128.45 86.55 135.6 75.73"/><circle class="cls-9" cx="135.6" cy="75.73" r="2.65"/><circle class="cls-9" cx="54.92" cy="102.43" r="2.65"/><path class="cls-10" d="M51.69,75.73h87.62"/></svg>
```
Comment (jy-binary) [inline src/images/pages/charting/tradingview-g-app.svg]: Negar, please write on the blackboard one hundred times: "I will not forget about caching"...

Comment (negar-binary) [inline src/images/pages/charting/tradingview-g-app.svg]: @jy-binary these files were not part of our appcache, I have just added them with this PR

Comment (jy-binary) [inline src/images/pages/charting/tradingview-g-app.svg]: That's not my point. Don't you need to change the file names of the images?

Comment (negar-binary) [inline src/images/pages/charting/tradingview-g-app.svg]: Sure, I'll make the change.

**Pull request merged**